configs/domain/index.ts

```diff
@@ -0,0 +1,21 @@
+import {getDenominationSymbols} from "./get-denomination-symbols";
+import {getLendingProtocols} from "./get-lending-protocols";
+import {getNetworks} from "./get-networks";
+import {getTokensByNetwork} from "./get-tokens-by-network";
+import {getTokensByTokenSymbol} from "./get-tokens-by-token-symbol";
+
+export default function () {
+    /**
+     * Goal is to capture domain specific configurations that is common
+     * across all Summer.fi systems and apps.
+     *
+     * Possible extensions could include Products / Product categories / Strategies etc
+     */
+    return {
+        tokens: getTokensByTokenSymbol(),
+        tokensByNetwork: getTokensByNetwork(),
+        lendingProtocols: getLendingProtocols(),
+        networks: getNetworks(),
+        denominationSymbols: getDenominationSymbols()
```

Comment:
this also returns labelByDenominationSymbol most probably 
@marcinciarka could you confirm?

Reply:
Only relevant to Subgraph that one

Reply:
That one should just be `denominationSymbols`. It's simply an Enum of denomination symbols converted to an object.

Reply:
I think labelByDemoniationSymbol might be misleading. And, that getDenominationSymbols makes more sense here.

It's converted the DemonationSymbol enum to an object.

Reply:
I feel the same way about getNetworks but have updated it. Interested to see what other folks think

Reply:
> I think labelByDemonationSymbol might be misleading. And, that getDenominationSymbols makes more sense here.
> 
> It's converted the DemonationSymbol enum to an object. That's all


I don`t understand why it might be misleading. It converted enum to a dictionary thus creating a new data structure which is different from enum in purpose so naming it the same is already misleading.
I'm missing the context of usage and purpose of this, could you clarify please?